wasi/wasi.go

```diff
@@ -254,6 +267,11 @@ func (w *WASIEnvironment) path_open(ctx *wasm.HostFunctionCallContext, fd, dirFl
 	return ESUCCESS
 }
 
+func (w *WASIEnvironment) fd_seek(ctx *wasm.HostFunctionCallContext, fd uint32, offset uint64, whence uint32, nwrittenPtr uint32) (err Errno) {
+	// not implemented yet
```

Comment:
@mathetake @nullpo-head I wonder if we shouldn't panic instead on all the stubbed functions? ex.

```suggestion
	panic("TODO: implement fd_seek")
```

Reply:
ENOSYS indicates that the host is not implementing this system call, so I think we don't need to panic!